src/Compilers/CSharp/Portable/Symbols/NativeIntegerTypeSymbol.cs

```diff
@@ -334,6 +334,8 @@ internal override int CalculateLocalSyntaxOffset(int localPosition, SyntaxTree l
             throw ExceptionUtilities.Unreachable;
         }
 
+        internal override bool IsNullableAnalysisEnabled() => false;
```

Comment:
Why is this returning `false` but the same override on `FunctionPointerMethodSymbol` throws? Not clear to me why one is reachable and the other is not.